diff --git b/app/node_modules/ripple-emulator/lib/server/emulate/hosted.js a/app/node_modules/ripple-emulator/lib/server/emulate/hosted.js
index 515262f..44cd4e8 100644
--- b/app/node_modules/ripple-emulator/lib/server/emulate/hosted.js
+++ a/app/node_modules/ripple-emulator/lib/server/emulate/hosted.js
@@ -124,6 +124,17 @@ function remoteInjection(opts) {
 }
 
 function localInjection() {
+
+    var cordovaPrepare;
+
+    try {
+        cordovaPrepare = require.resolve('cordova') ? require('cordova/src/prepare') : null;
+    }
+    catch (e) {
+        console.log('INFO: '.green +
+            'Could not find cordova as a local module. Expecting to find it installed globally.');
+    }
+
     function inject(file, req, res) {
         fs.readFile(file, "utf-8", function (err, data) {
             if (err) { throw new Error(err); }
@@ -161,23 +172,36 @@ function localInjection() {
     }
 
     return function (req, res, next) {
+
+        var callback = function (err) {
+            if (err) {
+                console.error('Failed to execute command: cordova prepare. '.red +
+                    'Make sure you\'ve installed cordova.'.red);
+            }
+
+            // make ripple compatible with phonegap
+            // as of version 3.0 phonegap uses phonegap.js instead of cordova.js
+            // but the files are identical
+            if (req.query.phonegap) {
+                var path = './platforms/' + req.staticPlatform + '/assets/www';
+                fs.readFile(path + '/cordova.js', function(err, data) {
+                    if(err) throw err;
+                    console.log('... copying cordova.js to phonegap.js');
+                    fs.writeFileSync(path + '/phonegap.js', data);
+                });
+            }
+            console.log('... done.');
+            handle(req, res, next);
+        };
+
         if (req.query.enableripple && req.staticPlatform) {
             console.log('refreshing project (platform: ' + req.staticPlatform + ') ...');
-            exec('cordova prepare ' + req.staticPlatform, function () {
-                // make ripple compatible with phonegap
-                // as of version 3.0 phonegap uses phonegap.js instead of cordova.js
-                // but the files are identical
-                if(req.query.phonegap) {
-                    var path = './platforms/' + req.staticPlatform + '/assets/www';
-                    fs.readFile(path + '/cordova.js', function(err, data) {
-                        if(err) throw err;
-                        console.log('... copying cordova.js to phonegap.js');
-                        fs.writeFileSync(path + '/phonegap.js', data);
-                    });
-                }
-                console.log('... done.');
-                handle(req, res, next);
-            });
+            if (!!cordovaPrepare) {
+                cordovaPrepare({ platforms: [ req.staticPlatform ] }).then(callback).done();
+            }
+            else {
+                exec('cordova prepare ' + req.staticPlatform, callback);
+            }
         }
         else {
             handle(req, res, next);
