diff --git a/app/node_modules/cordova/src/util.js b/app/node_modules/cordova/src/util.js
index ebc437d..5c31998 100644
--- a/app/node_modules/cordova/src/util.js
+++ b/app/node_modules/cordova/src/util.js
@@ -19,7 +19,8 @@
 var fs            = require('fs'),
     path          = require('path'),
     CordovaError  = require('./CordovaError'),
-    shell         = require('shelljs');
+    shell         = require('shelljs'),
+    optimist      = require('optimist');
 
 // Global configuration paths
 var HOME = process.env[(process.platform.slice(0, 3) == 'win') ? 'USERPROFILE' : 'HOME'];
@@ -125,7 +126,7 @@ exports = module.exports = {
         return projectDir;
     },
     projectWww: function(projectDir) {
-        return path.join(projectDir, 'www');
+        return path.join(projectDir, optimist.string('www-path').argv['www-path'] || 'www');
     },
     projectConfig: function(projectDir) {
         var rootPath = path.join(projectDir, 'config.xml');
